litecord/blueprints/guild/members.py

```diff
@@ -40,6 +42,17 @@ async def get_guild_member(guild_id, member_id):
     member = await app.storage.get_single_member(guild_id, member_id)
     return jsonify(member)
 
+@bp.route("/<int:guild_id>/members/<int:member_id>", methods=["PUT"])
+async def add_guild_member(guild_id, member_id):
+    """Forcibly add a member to a guild"""
+    await admin_check()
+
+    # TODO: if we ever support bots we will need to use checks for all of these here but 
+    # for now since this is only for the admin panel we can safely skip all checks
+    await add_member(guild_id, member_id, skip_check=True, basic=True)
+
+    return "", 204
```

Comment:
As per documentation, this should return 204 only if the member is already in the guild, else should be a 201 with the member object.